operator/apis/machinelearning/v1/seldondeployment_types.go

```diff
@@ -394,18 +397,19 @@ type SeldonDeploymentStatus struct {
 	Description      string                      `json:"description,omitempty" protobuf:"string,2,opt,name=description"`
 	DeploymentStatus map[string]DeploymentStatus `json:"deploymentStatus,omitempty" protobuf:"bytes,3,opt,name=deploymentStatus"`
 	ServiceStatus    map[string]ServiceStatus    `json:"serviceStatus,omitempty" protobuf:"bytes,4,opt,name=serviceStatus"`
+	DefaultReplicas  int32                       `json:"defaultReplicas,omitempty" protobuf:"string,5,opt,name=defaultReplicas"`
 }
 
 // +genclient
 // +genclient:noStatus
 // +kubebuilder:object:root=true
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 // +kubebuilder:storageversion
-
 // SeldonDeployment is the Schema for the seldondeployments API
 // +k8s:openapi-gen=true
 // +kubebuilder:resource:shortName=sdep
 // +kubebuilder:subresource:status
+// +kubebuilder:subresource:scale:specpath=.spec.defaultReplicas,statuspath=.status.defaultReplicas
```

Comment:
It's crazy that this is all you need to add support for `scale`!

Reply:
CRD power!